MarvelCharsApp/MarvelCharsApp/ViewModels/SideMenuViewModel.swift

```diff
@@ -42,6 +42,10 @@ class SideMenuViewModel {
 }
 
 extension SideMenuViewModel: MarvelRepositoryDelegate {
+    func didFetchMovies(movie: MovieModel) {
+        
```

Comment:
you should not add empty methods, if the method is repeating too much, shouldn't be on the delegate, or you should use another delegate